Classes/GraphQL/Resolver/Type/MutationResolver.php

```diff
@@ -379,7 +386,7 @@ public function uploadFile($_, array $variables): array
             $resource->setMediaType($file->getClientMediaType());
 
             if ($this->assetRepository->findOneByResourceSha1($resource->getSha1())) {
-                $result = 'EXISTS';
```

Comment:
I think I wanted a constant here to make the result parseable and the client could translate it, have to check my memory again what the intent was.

Reply:
OK, let me know. I didn't build the UI myself, so I don't know what I can contribute here, but we leave it open for the moment.

Reply:
Yes these constants should be translated in the JS part, see https://github.com/Flowpack/media-ui/blob/9b6c48c5992390574babded3a5d7dbd249f6982e/Resources/Private/JavaScript/asset-upload/src/components/FilePreview.tsx#L18